source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -154,6 +154,23 @@ void BioXASAppController::onRegionOfInterestBoundingRangeChanged(AMRegionOfInter
 		xasConfiguration_->setRegionOfInterestBoundingRange(region);
 }
 
+void BioXASAppController::goToBeamStatusView(AMControl *control)
```

Comment:
I would say this function name is descriptive enough.  If we think there is a better name at a later date we can change it then.